Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -583,18 +586,18 @@ PipelineProgram* ProgramShaderCache::GetPipelineProgram(const GLVertexFormat* ve
     if (!s_is_shared_context && vao != s_last_VAO)
       glBindVertexArray(s_last_VAO);
 
-    if (!ProgramShaderCache::CheckProgramLinkResult(
-            prog->shader.glprogid, vertex_shader ? &vertex_shader->GetSource() : nullptr,
-            geometry_shader ? &geometry_shader->GetSource() : nullptr,
-            pixel_shader ? &pixel_shader->GetSource() : nullptr))
+    if (!CheckProgramLinkResult(prog->shader.glprogid,
+                                vertex_shader ? vertex_shader->GetSource() : std::string_view{},
+                                geometry_shader ? geometry_shader->GetSource() : std::string_view{},
+                                pixel_shader ? pixel_shader->GetSource() : std::string_view{}))
```

Comment:
Guessing the `std::string_view{}` (vs. just `{}`) is necessary because `GetSource()` doesn't return a string-view (and the ternary would change the type being atuomatically determined)?